app/views/user_authentication/sign_up.html.erb

```diff
@@ -1,58 +1,74 @@
-<div class="container min-vh-100 d-flex justify-content-center align-items-center ">
+
+<div class="container-fluid min-vh-50 d-flex justify-content-center align-items-center">
 
 <form action="/insert_user" method="post">
 
-  <div class="mb-4">
-    <label for="email_box" class="form-label">Email</label>
+  <div class="mb-4 ">
+    <label for="email_box " class="form-label">Email</label>
 
-    <input id="email_box" class="form-control" name="query_email" id="exampleInputEmail1" placeholder="Choose an email..." type="text" aria-describebdy="emailHelp">
+    <input id="email_box" class="form-control border-green" name="query_email" id="exampleInputEmail1" placeholder="Choose an email..." type="text" aria-describebdy="emailHelp">
     <div id="emailHelp" class="form-text"></div>
   </div>
                  
   <div class="mb-4">
     <label for="first_name_box" class="form-label">First name</label>
       
-    <input type="text" class="form-control" placeholder="Enter First Name..."id="first_name_box" name="query_first_name">
+    <input type="text" class="form-control border-green" placeholder="Enter First Name..."id="first_name_box" name="query_first_name">
   </div>
          
   <div class="mb-4">
     <label for="last_name_box" class="form-label">Last name</label>
       
-    <input type="text" placeholder="Enter Last Name..."class="form-control" id="last_name_box" name="query_last_name">
+    <input type="text" placeholder="Enter Last Name..."class="form-control border-green" id="last_name_box" name="query_last_name">
   </div>
          
   
   <div class="mb-4"> 
-    <label for="last_name_box" class="form-label">Select Cohort</label>
-    <select class="form-select" aria-label="Default select example">
-    <option selected>Select</option>
-    <option value="1">1 - September 21st </option>
-    <option value="2">2 - January 30th </option>
-    <option value="3">3 - February 27th</option>
-    </select>
+    <label for="cohort_box" class="form-label">Select Cohort</label>
+
+    <input id="cohort_box" class="form-control border-green" name="query_cohort_id" placeholder="Choose a cohort..." type="cohort">
```

Comment:
Make this a drop down menu of the cohort IDs so they can be choosen when signing up

Reply:
fixed